### DISCUSSION TITLE
字数统计在哪里看？

### DISCUSSION THREAD

**Discussion by linelee02** [sjtug/SJTUThesis#880] [提问] [answered]
compile后，字数统计哪里可以看到？

**Comment by LogCreative** [comment, marked as answer]
参见 README：macOS 和 Linux 用户在终端输入

https://github.com/sjtug/SJTUThesis/blob/838c0e99dff5105c92ac08d5b7ed3f10c0318e1c/README.md?plain=1#L70

Windows 用户在命令行输入

https://github.com/sjtug/SJTUThesis/blob/838c0e99dff5105c92ac08d5b7ed3f10c0318e1c/README.md?plain=1#L81

**Reply by linelee02** [reply]
用的Windows。
抱歉还是不明白要怎么操作能看到wordcount，compile之后就结束了，没看到有显示字数统计。

**Reply by LogCreative** [reply]
在文件夹中右键“在终端中打开”：
![image](https://user-images.githubusercontent.com/61653082/210063555-cb5d3fd2-f796-4a45-bc35-686e085fc293.png)

输入 `.\Compile.bat wordcount` 后，回车
![image](https://user-images.githubusercontent.com/61653082/210063511-02118ca1-1eb0-4d58-a4e1-92c63640e011.png)
